cfme/tests/webui/test_general_ui.py

```diff
@@ -509,6 +529,53 @@ def test_provider_details_page_refresh_after_clear_cookies(
     assert login_view.is_displayed
 
 
+@pytest.mark.tier(1)
+@pytest.mark.provider([VMwareProvider], selector=ONE_PER_TYPE)
+@pytest.mark.parametrize("option", ALL_OPTIONS)
+def test_infrastructure_provider_left_panel_titles(
+    setup_provider, provider, option, soft_assert
+):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Infra
+        caseimportance: low
+        initialEstimate: 1/18h
+        testSteps:
+            1. Add an infrastructure provider and navigate to it's Details page.
+            2. Select Properties on the panel and check all items, whether they have their titles.
+            3. Select Relationships on the panel and check all items,
+                whether they have their titles.
+        expectedResults:
+            1.
+            2. Properties panel must have all items and clicking on each item should display
+                the correct page.
+            3. Relationships panel must have all items and clicking on each item should display
+                the correct page.
+    """
+    view = navigate_to(provider, "Details")
+    accordion = getattr(view.entities.sidebar, option)
+
+    for panel in ALL_OPTIONS[option]:
+        # Sometimes the panel doesn't enable immediately,
+        # using wait_for to give it some time to enable
+        wait_for(
+            lambda: not accordion.tree.is_disabled(partial_match(panel)),
+            fail_func=view.browser.refresh,
+            num=40,
+            delay=2,
+            message="Wait until the panel enables",
+            handle_exception=True,
+            silent_failure=True
+        )
+        if not accordion.tree.is_disabled(partial_match(panel)):
+            accordion.tree.select(partial_match(panel))
+            test_view = provider.create_view(ALL_OPTIONS[option][panel], wait="60s")
+            soft_assert(test_view.is_displayed, f"{test_view} not displayed.")
+        else:
+            logger.info(f"'{panel}' was not tested, it did not enable.")
```

Comment:
I think this should be a pytest.skip or pytest.fail - the item was expected to enable why wouldn't that be a failure?

Reply:
There are few items that get enabled once the provider receives data, but there are items such as `Timelines` that take some 20-30 minutes to enable itself, so I decided to just log it, I don't want to fail/skip the test in this case. Is there a way to quickly enable Timelines?

Reply:
I'm not aware of a way to quickly enable timelines in this case.

I still am not comfortable with this result for the test though.  There will be no indication in the results or artifactor report that this occurred - you would have to inspect the cfme.log file on every test execution to check for this message and to know whether or not timelines was actually tested.

A pytest xfail may be your best option here - that its more or less and expected failure state. We need some indication in results that there was missing coverage for this test case.

That or test timelines separately, putting more investigation into how to enable the option quickly.

Reply:
So John helped me trigger Timelines, activities like creating/deleting/reconfiguring VMs create an event and triggers Timelines. So I did that, I create VM as a part of setup.